.github/workflows/xtest.yml

```diff
@@ -60,8 +60,5 @@ jobs:
           export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
             | python3 -c "import sys, json; sys.stdout.write([tag for tag in json.load(sys.stdin)['Tags'] if not tag.endswith('.sig')][-1])")
           echo "Testing Backend [$BACKEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY
-          export FRONTEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/abacus \
```

Comment:
@ivanovSPvirtru removed it here because Frontend isn't used for the X-Test (frontend=false in correspondent tiltfile)